.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,123 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  push:
+    branches:
+      - master
```

Comment:
We need tests to run on Greenkeeper branches in order to check for breaking changes in fix/feature updates of our depenedncies. And we don't need to run tests on master, as we run them on pull requsets already

```suggestion
      - "greenkeeper/**"
```

Reply:
Since branches don’t have to be up to date at merge time, it’s possible to introduce issues through the merge. Eg one PR could introduce a change in lint config, and another PR could violate it. And it’s not just lint, it could be any kind of incompatibility between the two PRs. Running tests again on master is a good idea because it makes it easier to track down when the failures start and prevents us from accidentally releasing something that’s broken.

Reply:
We have this enabled, so we should be fine?

<img width="710" alt="image" src="https://user-images.githubusercontent.com/39992/73893227-e6bb8a80-482d-11ea-8d45-fcb0b77baa16.png">

Reply:
Oh! I didn't realize that. I guess I'm okay with it then!